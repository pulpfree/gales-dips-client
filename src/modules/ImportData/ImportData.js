import React, { Component } from 'react'
import PropTypes from 'prop-types'

import Divider from '@material-ui/core/Divider'
import Paper from '@material-ui/core/Paper'
import Typography from '@material-ui/core/Typography'
import { withStyles } from '@material-ui/core/styles'

import Header from '../Header/Header'
import ImportForm from './ImportForm'
import { styles as ms } from '../../styles/main'


class ImportData extends Component {

  render() {

    const { classes } = this.props

    return (
      <div>
        <Header />
        <Paper className={classes.paper}>
          <Typography
              gutterBottom
              variant="headline"
          >
            Import Sales Data
          </Typography>
          <Divider /><br />
          <ImportForm />
        </Paper>
      </div>
    )
  }
}
ImportData.propTypes = {
  classes: PropTypes.object.isRequired,
}

export default withStyles(ms)(ImportData)