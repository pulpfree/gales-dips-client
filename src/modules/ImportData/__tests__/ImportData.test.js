/* eslint no-undef: "off" */

import React from 'react'
import { shallow } from 'enzyme'

import ImportData from '../ImportData'

describe('ImportData', () => {

  let mountedC
  let props
  const reportC = () => {
    if (!mountedC) {
      mountedC = shallow(
        <ImportData {...props} />
      )
    }
    return mountedC
  }

  beforeEach(() => {
    props = {
      history: {location: {pathname: '/'}},
    }
    mountedC = undefined
  })

  it('matches snapshot', () => {
    expect(reportC()).toMatchSnapshot()
  })

})