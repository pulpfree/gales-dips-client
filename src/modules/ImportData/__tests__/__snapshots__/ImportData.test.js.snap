// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportData matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(ImportData)
    history={
        Object {
            "location": Object {
              "pathname": "/",
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "flex": "ImportData-flex-2",
        "mainContainer": "ImportData-mainContainer-3",
        "paper": "ImportData-paper-5",
        "popperClose": "ImportData-popperClose-6",
        "root": "ImportData-root-1",
        "titleLink": "ImportData-titleLink-4",
      },
      "history": Object {
        "location": Object {
          "pathname": "/",
        },
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "flex": "ImportData-flex-2",
          "mainContainer": "ImportData-mainContainer-3",
          "paper": "ImportData-paper-5",
          "popperClose": "ImportData-popperClose-6",
          "root": "ImportData-root-1",
          "titleLink": "ImportData-titleLink-4",
        },
        "history": Object {
          "location": Object {
            "pathname": "/",
          },
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
